tests/subscriptions/test_scheduler_processing_strategy.py

```diff
@@ -747,7 +747,7 @@ def test_produce_stale_message() -> None:
             Tick(
                 0,
                 offsets=Interval(3, 4),
-                timestamps=Interval(now - timedelta(seconds=50), now),
+                timestamps=Interval(now - timedelta(seconds=30), now),
```

Comment:
if this number is important, a comment saying why may be helpful to the next reader

Reply:
I think i was wrong about this being the issue, pushed a better fix now